clients/src/main/java/org/apache/kafka/common/record/Compressor.java

```diff
@@ -201,16 +201,24 @@ public void put(final byte[] bytes, final int offset, final int len) {
         }
     }
 
-    public void putRecord(long timestamp, byte[] key, byte[] value, CompressionType type, int valueOffset, int valueSize) {
+    /**
+     * @return CRC of the record
```

Comment:
As @hachikuji suggested, should we be consistent in how we describe the checksum (both in comments and variable names)? It's a nitpick though.